e2e/commands/start/start_test.go

```diff
@@ -15,26 +15,35 @@
 package start_test
 
 import (
+	"fmt"
+	"os"
 	"strings"
 
+	"github.com/ZupIT/horusec/internal/utils/testutil"
 	"github.com/google/uuid"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"github.com/onsi/gomega/gbytes"
 	"github.com/onsi/gomega/gexec"
-
-	"github.com/ZupIT/horusec/internal/utils/testutil"
 )
 
 var _ = Describe("running binary Horusec with start parameter", func() {
 	var (
-		session           *gexec.Session
-		err               error
-		flags             map[string]string
-		repoAuthorization string
-		configFilePath    = testutil.GoExample1
+		session                 *gexec.Session
+		err                     error
+		flags                   map[string]string
+		repoAuthorization       string
+		projectPath             = testutil.GoExample1
+		certificateFileWithPath string
 	)
 
+	BeforeSuite(func() {
+		file, err := os.CreateTemp(os.TempDir(), "*.crt")
+		if err != nil {
+			fmt.Print(err)
```

Comment:
We should fail the test with some errors occurs.

Reply:
adjusted!